### PR TITLE
What is COCO ? Issue #7

### PR DESCRIPTION
**Pull request by tarushi98**

### CODE REVIEW

File: project_2_image_captioning_project/cocoapi/README.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+4) Categories and Super Categories: There are a total of 91 categories and 11 Super Categories which were assigned by the researchers.The categories are usually the most commonly described objects by humans.
+
+For more information , read the paper here: https://arxiv.org/pdf/1405.0312.pdf
+To get the dataset , visit: http://cocodataset.org/
+
+
+
+## COCO API GUIDE
+
+This package provides Matlab, Python, and Lua APIs that assists in loading, parsing, and visualizing the annotations in COCO. Please visit http://cocodataset.org/ for more information on COCO, including for the data, paper, and tutorials. The exact format of the annotations is also described on the COCO website. The Matlab and Python APIs are complete, the Lua API provides only basic functionality.
```
Comment (vidhi-mody) [inline project_2_image_captioning_project/cocoapi/README.md]: Can you add the [links using markdown syntax](https://www.markdownguide.org/basic-syntax/#links)?

**Review by vidhi-mody — CHANGES_REQUESTED** [review]
Good work! I've left an inline comment

File: project_2_image_captioning_project/cocoapi/README.md (hunk 1 above)
Comment (tarushi98) [reply project_2_image_captioning_project/cocoapi/README.md]: Yes , will do ! :)

**Review by vidhi-mody — APPROVED** [review]
Good work

**Comment by tarushi98** [comment]
@vidhi-mody Thank you !! 😊

**Pull request merged**